src/components/Modals/BusinessHoursModal/BusinessHoursModalMain/BusinessHoursModalMain.tsx

```diff
@@ -12,9 +12,14 @@ const BusinessHoursModalMain = () => {
     return (
         <div className='lg:p-0 p-[1.6rem] h-full flex flex-col justify-between'>
             <div>
-                <Text size='sm'>
-                    <Localize i18n_default_text='Choose when you’re available to accept orders. Your ads will only be visible during these times.' />
-                </Text>
+                <div className='flex flex-col gap-3'>
+                    <Text size='sm'>
+                        <Localize i18n_default_text='Choose when you’re available to accept orders. Your ads will only be visible during these times.' />
+                    </Text>
+                    <Text size={textSize}>
+                        <Localize i18n_default_text='Business hours are displayed in your local time' />
```

Comment:
don't we need a period here?
```suggestion
                        <Localize i18n_default_text='Business hours are displayed in your local time.' />
```